build.fsx

```diff
@@ -133,25 +145,61 @@ Target "Pack" (fun _ ->
             ] |> String.concat " "
         "pack src/"+project+"/"+project+".fsproj -c "+ configuration + " -o ../../bin " + packParameters
         |> DotNetCli.RunCommand id
-    
+
     pack "Fantomas"
     pack "Fantomas.Cmd"
 )
 
+
+
+Target "TestExternalProjects" (fun _ ->
+    for project in externalProjectsToTest do
+        let relativeProjectDir = sprintf "external-project-tests/%s" project.DirectoryName
+        let buildCommandScript = sprintf "%s/%s/build.cmd" __SOURCE_DIRECTORY__ relativeProjectDir
+        if Fake.FileSystemHelper.directoryExists relativeProjectDir then
+            Fake.Git.Reset.ResetHard relativeProjectDir
+            Fake.Git.Branches.pull relativeProjectDir "origin" "master"
```

Comment:
Should we not build a stable release/tag instead? Then it is always reproducible.
Perhaps add this to the configuration.

Reply:
Yes we should.  I'll mark this as ready for review and then please make this adjsutment